test/rpc/test_rpc_call.py

```diff
@@ -100,6 +100,33 @@ def test_call_raises_on_invalid_calldata(deploy_info):
     }
 
 
+@pytest.mark.usefixtures("run_devnet_in_background")
+def test_call_raises_on_invalid_calldata_datatype_calldata_should_be_empty(deploy_info):
```

Comment:
The name of this test is not clear to me and it is quite long. You can also rely on the function docstring.

Reply:
gotcha!